plugins/catalog-import/src/components/ImportComponentPage.tsx

```diff
@@ -14,36 +14,29 @@
  * limitations under the License.
  */
 
-import React, { useState } from 'react';
-import { Grid, Typography } from '@material-ui/core';
+import { Entity } from '@backstage/catalog-model';
 import {
-  InfoCard,
-  Page,
+  ConfigApi,
+  configApiRef,
   Content,
+  ContentHeader,
   Header,
+  InfoCard,
+  Page,
   SupportButton,
-  ContentHeader,
-  RouteRef,
   useApi,
-  configApiRef,
-  ConfigApi,
 } from '@backstage/core';
-import { RegisterComponentForm } from './ImportComponentForm';
-import ImportStepper from './ImportStepper';
-import ComponentConfigDisplay from './ComponentConfigDisplay';
-import { ImportFinished } from './ImportFinished';
-import { PartialEntity } from '../util/types';
+import { Grid, Typography } from '@material-ui/core';
+import React from 'react';
+import { ImportStepper } from './ImportStepper';
+import { StepperProviderOpts } from './ImportStepper/defaults';
 
 export type ConfigSpec = {
   type: 'tree' | 'file';
   location: string;
-  config: PartialEntity[];
+  config: Entity[];
```

Comment:
This type should actually be removed because it is a remnant of the old code that I missed to delete.

I can add the `PartialEntity` back and use it at the respective places.

Reply:
That being said, this type is a bit too partial for my taste, since it also means that `kind`, `namespace` (could fallback to `default`), and `name` are suddenly optional fields and requires either null-checks or `!`'s in the code.